HW2.3.java

```diff
@@ -1 +1,23 @@
+package com.company;
 
+import java.util.Scanner;
+
+public class MathExpression {
+    final double epsilon = 0.0001;
```

Comment:
add access modifier and make it constant:
private static final......

Reply:
Ok